Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.py

```diff
@@ -1347,6 +1352,47 @@ def fetch_incidents():
     demisto.setLastRun({'time': current_run.isoformat().split('.')[0]})
 
 
+def add_capitalized_hash(threats_context):
```

Comment:
Please improve the name of this method, it is not very self-explanatory.
I would suggest something like add_capitalized_hash_to_context.

Also, I think you can handle both cases in one method, see my suggestion

Reply:
Done

Reply:
Done